canva.yaml

```diff
@@ -0,0 +1,160 @@
+name: Canva
+description: |
+  ## Overview
+  A Model Context Protocol (MCP) server for Canva. Provides comprehensive, real-time Canva development context and tools to LLMs to help you build on Canva faster.
+
+  ## Features
+  - **Designs**: Search your Canva designs (docs, presentations, whiteboards, videos, sheets, etc.) by title or content.
+  - **Extract**: View and extract content from Canva docs, presentations, and more.
+  - **File Management**: List, create, and organize folders in your Canva account.
+  - **Export**: Export designs to formats like PDF, PNG, JPG, PPTX, GIF, and MP4 (if supported).
```

Comment:
given we have no local filesystem supports yet, does `export` tools work?

Reply:
Oh actually the export tools just provide a URL that you can use to download the files. So we are good.

Reply:
oh nice